app/views/recipe_foods/_recipe_food.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<tr id="<%= dom_id recipe_food %>">
+  <td><%= recipe_food.food.name %></td>
+  <td><%= recipe_food.quantity %><%= recipe_food.food.measurement_unit %></td>
+  <td>$<%= recipe_food.food.price * recipe_food.quantity %></td>
+  <td>
+    <%= button_to(
+      "Delete ingredient",
+      recipe_food_path(recipe_food.id),
+      method: :delete,
+      class: "btn btn-outline-danger"
+    ) %>
+  </td>
+</tr>
```

Comment:
- [ ] I suggest adjusting the layout of your recipe show page to match the wireframe. Currently, elements for instance the add ingredient button and the public switch are not aligned with your design.
  
  ### Your design
  ![Screenshot from 2023-09-09 19-00-08](https://github.com/edahigure/Recipe-app/assets/92303891/ebe1ed0d-0f58-4bdc-a22f-205b6e3f22a6)
  
  ### Given Wireframe
  
  ![image](https://github.com/edahigure/Recipe-app/assets/92303891/2cfc9215-a943-4eeb-8e6d-4d7606045d05)